src/app/feed/feed-card/feed-card.component.ts

```diff
@@ -126,4 +132,28 @@ export class FeedCardComponent implements OnInit {
 		}
 
 	}
+
+	private sanitizeandembedURLs(links) {
+		links.forEach((link,i) => {
+			let videoid = links[i].match(/(?:https?:\/{2})?(?:w{3}\.)?youtu(?:be)?\.(?:com|be)(?:\/watch\?v=|\/)([^\s&]+)/);
+			if(videoid != null) {
+   				links[i] = "http://www.youtube.com/embed/" + videoid[1];
+			}
+			links[i] = this.sanitizer.bypassSecurityTrustResourceUrl(links[i]);
+		})
+	}
+
+		private imageURLs(links) {
+		let img1 = new RegExp('https:\\/\\/abs\\.twimg\\.com\\/');
```

Comment:
`https://pic.twitter.com/{{hashcode}}.jpg` Are these due to the scraping error from server?